### PR TITLE
Add rust:1.34 runtime support to default deployments

### PR DESCRIPTION
**Pull request by rdiaz82**
## Description
The included changes in this PR are the following:
### 1. Add rust:1.34 to core ansible/vagrant deploy
  - [X] Add rust:1.34entry into ansible/files/runtimes.json
  - [X] Add rust-actions.md link in docs/actions.md
  - [ ] Implement rust-actions.md documentation
  - [X] Update unicode tests https://github.com/apache/incubator-openwhisk/tree/master/tests/dat/actions/unicode.tests
  - [X] Update swagger file https://github.com/apache/incubator-openwhisk/blob/master/core/controller/src/main/resources/apiv1swagger.json
### 2. Add rust:1.34 to docker-compose deploy
  - [x]  Add rust:1.34 into runtime manifest
https://github.com/apache/incubator-openwhisk-devtools/blob/master/docker-compose/docker-whisk-controller.env#L64
### 3. Add rust:1.34 to deploy-kube
 -  [x] Add rust:1.34 into runtime manifest
https://github.com/apache/incubator-openwhisk-deploy-kube/blob/master/helm/openwhisk/runtimes.json
### 4. Update wskdeploy with rust:1.34
- [x] Update spec, and tests for rust:1.34
- [x] https://github.com/apache/incubator-openwhisk-wskdeploy/blob/master/runtimes/runtimes.go#L267
- [x] https://github.com/apache/incubator-openwhisk-wskdeploy/blob/master/specification/html/spec_actions.md#valid-runtime-names
- [x] https://github.com/apache/incubator-openwhisk-wskdeploy/blob/master/tests/src/integration/runtimetests/manifest.yaml#L5
### 5. Update website with rust:1.34
- [ ] Update js action section with rust:1.34
- [ ] https://github.com/apache/incubator-openwhisk-website/blob/master/_layouts/documentation.html
- [ ] https://github.com/apache/incubator-openwhisk-website/tree/master/_includes/code

## Related issue and scope
https://github.com/apache/incubator-openwhisk/issues/4443

## My changes affect the following components
- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [X] Deployment
- [ ] CLI
- [ ] General tooling
- [X] Documentation

## Types of changes
- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [X] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:
- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [X] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [X] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [X] I updated the documentation where necessary.

### CODE REVIEW

**Comment by rabbah** [comment]
@rdiaz82 is the readme all that's left on this?

**Comment by rdiaz82** [comment]
@rabbah are there still some files to modify. I'm going to try to do it asap because they are easy changes

**Comment by dgrove-oss** [comment]
@rdiaz82 -- what's the status on this PR?  Thanks.

**Comment by rdiaz82** [comment]
It is still in progress, right now it is implemented the tasks with the tick mark. I have had a lot of work lately and I haven't had time to finish it

**Comment by dgrove-oss** [comment]
thanks for the update.

**Comment by rabbah** [comment]
Hi @rdiaz82 - do you want some help on this PR? 

We have a new community member interested in helping out on Rust and is willing to take on one the tasks still to be done, namely documenting how to use the Rust runtime.

**Comment by rdiaz82** [comment]
Hey!! To be honest, I don't really have time to finish this, so if another member is interested on taking it I don't mind, sorry for taking too long to finish this!!

A couple of months ago I did some tasks of the list and here are the associated PRs:
https://github.com/apache/openwhisk-wskdeploy/pull/1074
https://github.com/apache/openwhisk-deploy-kube/pull/533
https://github.com/apache/openwhisk-devtools/pull/295

File: ansible/files/runtimes.json [hunk 1]
```diff
@@ -193,7 +193,24 @@
...
                 }
             }
+        ],
+        "rust": [
+            {
+                "kind": "rust:1.34",
+                "default": true,
+                "image": {
+                    "prefix": "openwhisk",
+                    "name": "actionloop-rust-v1.34",
```
Comment (dgrove-oss) [inline ansible/files/runtimes.json]: docker image is now being published as `action-rust-v1.34`

File: tests/dat/actions/unicode.tests/rust-1.34.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
+extern crate serde_json;
```
Comment (rabbah) [inline tests/dat/actions/unicode.tests/rust-1.34.txt]: License?

Comment (dgrove-oss) [reply tests/dat/actions/unicode.tests/rust-1.34.txt]: scancode is ignoring .txt files.   
most of the files in this directory are missing license headers 😿

Comment (rabbah) [reply tests/dat/actions/unicode.tests/rust-1.34.txt]: Oops.

Comment (dgrove-oss) [reply tests/dat/actions/unicode.tests/rust-1.34.txt]: added.

**Comment by dgrove-oss** [comment]
I plan to go ahead and merge this once travis clears the last license header fix I just committed to the branch.

**Pull request merged**